### DISCUSSION TITLE
二进制文件怎么打开？

### DISCUSSION THREAD

**Discussion by shanshui4731csy** [go-olive/olive#146] [Q&A]
点击后叫我cmd运行，怎么运行？

**Comment by misaka10843** [comment]
具体看你是什么系统
但是基本上都是需要`./olive`

**Comment by ColinSimpleee** [comment]
## 📋 nowalls 迁移项目详细开发方案

经过对项目现有架构的深入分析，现制定以下详细开发方案：

### 🎯 核心目标

1. **个人中心摊主版**：创建独立的个人中心页面类型，支持摊主信息展示
2. **招募活动列表组件**：在自定义页面中新增招募活动列表组件
3. **活动页面可配置化**：全部活动页改造为可配置页面（卡片样式、排序、筛选）
4. **后台模块整理**：摊主管理独立成新模块，数据表迁移
5. **租户隔离修复**：修复接口未区分租户的问题 ⚠️
6. **前端页面迁移**：迁移剩余页面并修复变量/路径问题

---

### 📅 开发阶段规划

#### 🔴 阶段一：基础设施和安全（第1-2周）

**任务5：修复租户隔离问题** ⭐ **最高优先级**
- 审计所有 API 接口的 tenant_id 使用情况
- 修复 `micro_get_openid` 接口
- 修复所有活动列表接口
- 创建自动化租户隔离测试
- 批量验证所有查询、更新、删除操作

**任务4：数据迁移和模块整理**
- 验证8张表的 tenant_id 完整性
- 创建独立的"摊主管理"后台模块
- 重组页面结构（从 `event/vendor_brand/` 迁移到 `vendor/`）
- 更新路由配置和菜单配置

#### 🟡 阶段二：核心功能实现（第2-4周）

**任务3：活动页面配置系统**
- 扩展现有配置表，添加活动页面配置字段
- 实现后台配置界面（卡片样式、排序、筛选）
- uni_app 活动列表页动态应用配置

**任务2：招募活动列表组件**
- 创建 RecruitmentListComponent 组件
- 实现属性面板
- 注册到 SimplePageBuilder
- 实现数据接口

#### 🟢 阶段三：体验优化（第4-5周）

**任务1：摊主个人中心**
- 创建 `vendor-center.vue` 页面
- 实现有/无身份状态UI
- 样式变量适配
- 集成到菜单配置系统

**任务6：前端页面迁移和优化**
- 迁移缺失页面（等级管理、任务管理、申请记录）
- 统一样式变量
- 修复硬编码路径
- 提取文案到配置表

---

### ⏱️ 时间估算

- **阶段一（任务5+4）**：2周（10个工作日）
- **阶段二（任务3+2）**：2周（10个工作日）
- **阶段三（任务1+6）**：1周（5个工作日）
- **测试和优化**：1周（5个工作日）
- **文档和培训**：0.5周（2-3个工作日）

**项目总周期**：6.5周（约32个工作日）

---

### ✅ 关键决策

根据团队讨论，已确认：

1. **开发优先级**：租户隔离修复作为最高优先级（安全第一）
2. **模块迁移方式**：直接迁移，不保留旧路径重定向
3. **数据库设计**：扩展现有配置表，不创建新表
4. **资源分配**：按阶段推进，确保每个阶段质量

---

### 📝 下一步行动

接下来将创建对应的 GitHub Issues 来跟踪每个任务的进度。每个 Issue 将包含：

- 详细的实施步骤
- 关键文件清单
- 测试验证清单
- 技术方案和代码示例

完整的技术方案文档已保存在项目计划文件中。
